PowerUp/app/src/main/java/powerup/systers/com/vocab_match_game/VocabMatchEndActivity.java

```diff
@@ -11,8 +11,7 @@
 import powerup.systers.com.R;
 import powerup.systers.com.ScenarioOverActivity;
 import powerup.systers.com.powerup.PowerUpUtils;
-import powerup.systers.com.vocab_match_game.VocabMatchSessionManager;
-import powerup.systers.com.sink_to_swim_game.SinkToSwimGame;
+
```

Comment:
Extra new line added. Can you remove it? Thanks!

Reply:
Done. Thanks!